### ISSUE TITLE
Running anchor test on anchor example returns error

### ISSUE THREAD

**Issue by asvsfs** [pyth-network/pyth-sdk-rs#114]
Steps to produce 

1. solana-keygen new -o program_address.json
2. cd examples/sol-contract-anchor
3. anchor build
4. anchor deploy --program-keypair program_address.json --program-name sol-anchor-contract
5. npm i
6. anchor test

> 1) sol-anchor-contract
       Check loan to value ratio.:
     Error: AnchorError caused by account: pyth_loan_account. Error Code: PythError. Error Number: 6005. Error Message: Pyth has an internal error..

**Comment by yafetful** [comment]
metoo

**Comment by blue-codes-yep** [comment]
Yeah I am receiving compatibility mismatches, with the solana-program version. 
Will update if I come to work around previously I resulted to just manually building and not using anchor.. 

Anchor just dropped an update today which caused me to have some other conflict issues as well, and I'd prefer Anchor stay up to date so.. 

If I remember was something like that following:
  cargo build-sbf --manifest-path programs/nexus-order/Cargo.toml
  Then using anchor idl build > ../../target/idl/{filename.json} in my programs folder.
